.maintain/chaostest/src/commands/singlenodeheight/singlenodeheight.js

```diff
@@ -0,0 +1,135 @@
+const {Command, flags} = require('@oclif/command')
+
+class SingleNodeHeightCommand extends Command {
+  async run() {
+    const {flags} = this.parse(SingleNodeHeightCommand)
+    const imageTag = flags.image || 'parity/substrate:latest'
+    const port = flags.port || 9933
+    const url = flags.url || 'http://localhost'
+    const wait = flags.wait || 600 * 1000
+    const namespace = flags.namespace || 'substrate-ci'
+    const serviceName = 'substrate-service'
+    const deployName = 'substrate-deployment' + process.env.CI_COMMIT_SHORT_SHA
+    const k8s = require('../../k8s')
+    const jsonRpc = require('../../utils/json-rpc')
+
+    async function loadTest() {
```

Comment:
I am a bit confused by the word `load`. This does neither _load_ something, e.g. from disc, nor _load-test_ something, e.g. spawn a lot of requests. Why not something like `startTest`?